runtime/syntax/python.vim

```diff
@@ -105,6 +105,7 @@ syn keyword pythonOperator	and in is not or
 syn keyword pythonException	except finally raise try
 syn keyword pythonInclude	from import
 syn keyword pythonAsync		async await
+syn keyword pythonClassVar	self cls
```

Comment:
I wouldn't put them in this group of real keywords because they wouldn't be in the output of the command above in the comment (line 95).
Perhaps moving them _below_ soft keywords with an appropriate comment that they are _de facto_ convention, rather than the real keywords. After all, we could call them `Bob` and `Fred` and the result would be the same. We don't do that because it's not a convention and wouldn't be nice to the code maintainers. 🙂

Reply:
Good idea. I relocated them with a comment in the latest commit ([b9e5404](https://github.com/vim/vim/pull/17968/commits/b9e5404cfd2ba160bc0dab784370ed8ea8249fe2)).